force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -239,10 +282,13 @@ public with sharing class FlowControlLoopsCollections {
     public static Set<String> createStringSet(List<String> inputList) {
         // Create a new Set of strings
         Set<String> stringSet = new Set<String>();
-
-        //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+        for (String input : inputList) {
+            if (stringSet.contains(input) ) {
+                continue;
+            }
+            stringSet.add(input);
+        }
+        return stringSet; 
```

Comment:
1. This is the correct use of the contains statement. A lot of students get that wrong.
2. You don't need to check for membership on line 286. A set, by its definition, can not have duplicates. Add the same value to a set 100 times and see what it contains.
3. More advanced tip. You can pass a list directly into a set constructor. 

```
 public static Set<String> createStringSet(List<String> inputList) {
        return new Set<String>(inputList); // Replace null with the variable you used to store the result
    }
 ```